test/vm.install

```diff
@@ -22,22 +22,27 @@ printf "[WebService]\\nAllowUnencrypted=true\\n" > /etc/cockpit/cockpit.conf
 # disable session timeout as some tests wait for too long
 printf "[Session]\\nIdleTimeout=0\\n" >> /etc/cockpit/cockpit.conf
 
+cat /etc/cockpit/cockpit.conf
```

Comment:
Is this here for some reason or just for debugging?

Reply:
oh, good catch! It's just for debugging, because I found an error screenshoot to cockpit login page, so I just suspect the configuration `cockpit.conf` does not work.